### ISSUE TITLE
Fix FN S6249: 'import ... =' usage in typescript

### ISSUE THREAD

**Issue by pedro-oliveira-sonarsource** [SonarSource/SonarJS#3656]
### **What**

In  #3204 , support for _aws-cdk-lib.aws_s3.Bucket_ was added, where we raise an issue if a bucket is configured to allow unencrypted communications.

On a validation sprint, when analyzing the [code sample](https://github.com/SonarSource/security-expected-issues/pull/720/files) produced during the research for rule specification, FNs were detected due to how the s3 library was imported.

'import ... =' can only be used in TypeScript files.

**False-Negative**
```
import cdk = require('aws-cdk-lib');
import iam = require('aws-cdk-lib/aws-iam');
import s3 = require('aws-cdk-lib/aws-s3'); // <------------
^^^^^^^^

const enforceSSLFalse = new s3.Bucket(this, 'enforce.ssl.false', {
  bucketName: 'out.enforce.ssl.false',
  versioned: true,
  publicReadAccess: false,
  removalPolicy: cdk.RemovalPolicy.DESTROY,
  enforceSSL: false // FN - Sensitive: Make sure authorizing HTTP requests is safe here.
});
```
**True-Positive**
```
import cdk = require('aws-cdk-lib');
import iam = require('aws-cdk-lib/aws-iam');
const s3 = require('aws-cdk-lib/aws-s3'); // <------------
^^^^^^^

const enforceSSLFalse = new s3.Bucket(this, 'enforce.ssl.false', {
  bucketName: 'out.enforce.ssl.false',
  versioned: true,
  publicReadAccess: false,
  removalPolicy: cdk.RemovalPolicy.DESTROY,
  enforceSSL: false // TP - Sensitive: Make sure authorizing HTTP requests is safe here.
});
```

**Comment by victor-diez-sonarsource** [comment]
We need to add support `TSImportEqualsDeclaration` in [our FQN resolution](https://github.com/SonarSource/SonarJS/blob/master/eslint-bridge/src/linting/eslint/rules/helpers/module.ts#L120).

Supported constructions at this moment: https://github.com/SonarSource/SonarJS/issues/3469#issuecomment-1276390408

[Reference](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require)
